extensions/wikia/PortableInfobox/controllers/PortableInfoboxParserTagController.class.php

```diff
@@ -187,12 +193,32 @@ private function getThemeName( $params, $value ) {
 	}
 
 	private function getLayout( $params ) {
+		$validator = new InfoboxParamsValidator();
 		$layoutName = isset( $params[ 'layout' ] ) ? $params[ 'layout' ] : false;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "false" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1125)